.github/workflows/e2e.yml

```diff
@@ -19,6 +19,9 @@ jobs:
       matrix:
         os: ${{ fromJSON(inputs.platforms) }}
     steps:
+      - name: Output inputs
+        run: echo "${{ toJSON(inputs) }}"
+        shell: sh
```

Comment:
I ask for `sh` specifically because the `echo` in the default Windows shell prints the output with all kinds of ugly escapes.